plugins/db-mongo/static/package-json/package.template.json

```diff
@@ -0,0 +1,75 @@
+{
```

Comment:
it should be `package.json` and not `package.template.json`

Reply:
@abrl91 ok, now i see this generate the file with that name, i will change it :)